Author: Diane Trout <diane@ghic.org>
Description: loading python module linked to jemalloc shared library
  fails with "libjemalloc.so: cannot allocate memory in static TLS block"
  See https://github.com/jemalloc/jemalloc/issues/937
  The easy answer is to just use the system malloc instead.

--- a/python/pyarrow/tests/test_memory.py
+++ b/python/pyarrow/tests/test_memory.py
@@ -29,7 +29,7 @@
 
 possible_backends = ["system", "jemalloc", "mimalloc"]
 
-should_have_jemalloc = sys.platform == "linux"
+should_have_jemalloc = False   #sys.platform == "linux"
 should_have_mimalloc = sys.platform == "win32"
 
 
